starboard/shared/starboard/player/filter/filter_based_player_worker_handler.cc

```diff
@@ -132,7 +132,7 @@ HandlerResult FilterBasedPlayerWorkerHandler::Init(
       std::string error_message =
           FormatString("Required channel %d is greater than maximum channel %d",
                        required_audio_channels, supported_audio_channels);
-      return {false, std::move(error_message.c_str())};
+      return {false, error_message};
```

Comment:
Let's use `HandleResult{...}` here, so it more explicit on what we are returning here.